### PR TITLE
[CN-305] Add documentation for phone home

### PR DESCRIPTION
**Pull request by mtyazici**

### CODE REVIEW

**Review by hasancelik — APPROVED** [review]

File: docs/modules/ROOT/pages/phone-homes.adoc [hunk 1]
```diff
@@ -0,0 +1,45 @@
+= Phone Homes
+
+Hazelcast uses phone home data to learn about the usage of Hazelcast Platform Operator.
```
Comment (JakeSCahill) [inline docs/modules/ROOT/pages/phone-homes.adoc:3]:
```suggestion
:description: Each instance of Hazelcast Platform Operator reports usage data to the Hazelcast phone home server when a Custom Resource is successfully deployed and every 24 hours after that. Hazelcast uses this data to learn about the usage of Hazelcast Platform Operator.
```

File: docs/modules/ROOT/pages/phone-homes.adoc [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+Hazelcast Platform Operator instances call our phone home server for every Custom Resource's initial successful deployment and every 24 hours.
```
Comment (JakeSCahill) [inline docs/modules/ROOT/pages/phone-homes.adoc:5]:
```suggestion
{description}
```

File: docs/modules/ROOT/pages/phone-homes.adoc [hunk 3]
```diff
@@ -0,0 +1,45 @@
+= Phone Homes
+
```
Comment (JakeSCahill) [inline docs/modules/ROOT/pages/phone-homes.adoc:2]:
```suggestion
```

File: docs/modules/ROOT/pages/phone-homes.adoc [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+Hazelcast uses phone home data to learn about the usage of Hazelcast Platform Operator.
+
+Hazelcast Platform Operator instances call our phone home server for every Custom Resource's initial successful deployment and every 24 hours.
+
+**What is sent in?**
```
Comment (JakeSCahill) [inline docs/modules/ROOT/pages/phone-homes.adoc:7]:
```suggestion
What Information is Sent in a Phone Home?
```

File: docs/modules/ROOT/pages/phone-homes.adoc [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+The following information is sent in a phone home:
+
+* Operator version
+* K8s distribution, EKS, GKE, AKS etc.
+* K8s version
+* Operator uptime
+* UID of the operator
+* Operator installation origin, e.g Dockerhub, Red Hat Connect
```
Comment (JakeSCahill) [inline docs/modules/ROOT/pages/phone-homes.adoc:16]:
```suggestion
* Operator installation origin such as Dockerhub or Red Hat Connect
```

File: docs/modules/ROOT/pages/phone-homes.adoc [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+* Number of created OSS Hazelcast clusters
+* Number of created EE Hazelcast clusters
+* Number of created Management Centers
```
Comment (JakeSCahill) [inline docs/modules/ROOT/pages/phone-homes.adoc:19]:
```suggestion
* Number of created Management Center instances
```

File: docs/modules/ROOT/pages/phone-homes.adoc [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+The following information is sent in a phone home:
+
+* Operator version
+* K8s distribution, EKS, GKE, AKS etc.
+* K8s version
+* Operator uptime
+* UID of the operator
+* Operator installation origin, e.g Dockerhub, Red Hat Connect
+* Number of created OSS Hazelcast clusters
+* Number of created EE Hazelcast clusters
```
Comment (JakeSCahill) [inline docs/modules/ROOT/pages/phone-homes.adoc:18]:
```suggestion
* Number of created Hazelcast Enterprise clusters
```

File: docs/modules/ROOT/pages/phone-homes.adoc [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+
+The following information is sent in a phone home:
+
+* Operator version
+* K8s distribution, EKS, GKE, AKS etc.
+* K8s version
+* Operator uptime
+* UID of the operator
+* Operator installation origin, e.g Dockerhub, Red Hat Connect
+* Number of created OSS Hazelcast clusters
```
Comment (JakeSCahill) [inline docs/modules/ROOT/pages/phone-homes.adoc:17]:
```suggestion
* Number of created Hazelcast Open Source clusters
```

File: docs/modules/ROOT/pages/phone-homes.adoc [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+* Expose Externally specific:
+** Number of Hazelcast clusters where unisocket is used
+** Number of Hazelcast clusters where smart client is used
+** Number of Hazelcast clusters where LoadBalancer discovery is used
+** Number of Hazelcast clusters where NodePort discovery is used
+** Number of Hazelcast clusters where LoadBalancer member access is used
+** Number of Hazelcast clusters where node external IP member access is used
+** Number of Hazelcast clusters where node name member access is used
+
+**Disabling Phone Homes**
```
Comment (JakeSCahill) [inline docs/modules/ROOT/pages/phone-homes.adoc:32]:
```suggestion
== Disabling Phone Homes
```

File: docs/modules/ROOT/pages/phone-homes.adoc [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+
+You can disable the phone home using the environment variable `PHONE_HOME_ENABLED`.
```
Comment (JakeSCahill) [inline docs/modules/ROOT/pages/phone-homes.adoc:34]:
```suggestion
To disable phone homes, set the `PHONE_HOME_ENABLED` environment variable to `false`.
```

File: docs/modules/ROOT/pages/phone-homes.adoc [hunk 11]
```diff
@@ -0,0 +1,45 @@
...
+
+Simply add the following to the environment variable list in the deployment object in `bundle.yaml`.
```
Comment (JakeSCahill) [inline docs/modules/ROOT/pages/phone-homes.adoc:36]:
```suggestion
```

File: docs/modules/ROOT/pages/phone-homes.adoc [hunk 12]
```diff
@@ -0,0 +1,45 @@
...
+
+```yaml
+- name: PHONE_HOME_ENABLED
+  value: "false"
+```
+
+**Phone Home URL**
```
Comment (JakeSCahill) [inline docs/modules/ROOT/pages/phone-homes.adoc:43]:
```suggestion
== Phone Home URL
```

File: docs/modules/ROOT/pages/phone-homes.adoc (hunk 4 above)
Comment (JakeSCahill) [reply docs/modules/ROOT/pages/phone-homes.adoc:7]: It's better to use headings as these are indexed and appear in the ToC.

File: docs/modules/ROOT/pages/phone-homes.adoc (hunk 2 above)
Comment (JakeSCahill) [reply docs/modules/ROOT/pages/phone-homes.adoc:5]: Displays the contents of `description`

File: docs/modules/ROOT/pages/phone-homes.adoc (hunk 1 above)
Comment (mtyazici) [reply docs/modules/ROOT/pages/phone-homes.adoc:3]: :+1:

**Review by JakeSCahill — APPROVED** [review]

**Pull request merged**